systers_portal/blog/forms.py

```diff
@@ -84,3 +84,22 @@ class Meta:
         helper_class = SubmitCancelFormHelper
         helper_cancel_href = "{% url 'view_community_resource' " \
                              "community.slug object.slug %}"
+ 
+ 
+class TagForm(ModelFormWithHelper):
```

Comment:
`TagForm` should not appear as an addition in the pull request, that means you have messed up the git history. You should rebase against develop branch to have the latests commits on your branch. But since this pull request is old, you will have conflicts.